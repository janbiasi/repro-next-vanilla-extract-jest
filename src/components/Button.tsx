import { PropsWithChildren, forwardRef } from "react";
import { buttonStyle } from "./Button.css";

export const Button = forwardRef<HTMLButtonElement, PropsWithChildren>(({ children }, ref) => (
    <button ref={ref} className={buttonStyle({
        type: 'primary'
    })}>
        {children}
    </button>
));

Button.displayName = "Button";
