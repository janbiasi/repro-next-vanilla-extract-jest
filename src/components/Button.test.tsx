import { render, screen } from '@testing-library/react';
import { Button } from './Button';

describe('Components/Button', () => {
    it('some test case', () => {
        const { baseElement } = render(<Button>Button</Button>);
        expect(baseElement).toMatchSnapshot();
    })
})