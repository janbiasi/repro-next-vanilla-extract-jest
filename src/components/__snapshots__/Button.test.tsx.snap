// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components/Button some test case 1`] = `
<body>
  <div>
    <button>
      Button
    </button>
  </div>
</body>
`;
